tests/conftest.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from webui.settings import ConfigFile
+import webui
+import tempfile
+
+
+def set_pcw_ini(filename, add=''):
+    with open(filename, "w") as f:
+        f.write(add)
+
+
+@pytest.fixture(scope="session", autouse=True)
+def generalSetup():
+    tmpFile = tempfile.mkstemp()
+    webui.settings.CONFIG_FILE = tmpFile[1]
+    set_pcw_ini(tmpFile[1])
+    yield tmpFile[1]
```

Comment:
not sure about that yield here, can you explain why not `return` or yield in a loop?